packages/story-editor/src/components/panels/design/textStyle/karma/stylePresets.karma.js

```diff
@@ -56,7 +56,7 @@ describe('Panel: Style Presets', () => {
       await fixture.editor.library.textTab.click();
 
       // Verify that no styles are added currently.
-      const noStylesText = fixture.screen.getByText('No Styles Saved');
+      let noStylesText = fixture.screen.getByText('No Styles Saved');
       expect(noStylesText).toBeDefined();
```

Comment:
Why make this a let? Why just rename the second instance of `noStylesText`?

Reply:
Hm, is this a merging issue? For some reason, `// Verify that no styles are added currently.` is happening twice here (here and below), there's no reason to check for it twice.

Reply:
I'm guessing the merging issue came from before your PR 🙂 Somehow checking for `noStylesText` got in twice.

Reply:
think that was a bad rebase from when i updated sidebar from main on friday, thank you for fixing it.